contrib/ci/generate_debian.py

```diff
@@ -95,16 +97,25 @@ def update_debian_control(target):
     with open(control_in, 'r') as rfd:
         lines = rfd.readlines()
 
-    deps = parse_control_dependencies("build")
+    with open(control_qubes_in, 'r') as rfd:
+        qubes_lines = rfd.readlines()
```

Comment:
This file, maybe it can be simplified.  Suggestion:

```
if QUBES
    control_qubes_in = ....
    with open(control_qubes_in, 'r') as rfd:
        lines += rfd.readlines()
```

Isn't something like that enough?

Reply:
Implemented.